### PR TITLE
Use ```text for non-code blocks

### PR DESCRIPTION
**Pull request by Idloj**
This complements NetLogo/NetLogo#1165

@nicolaspayette, I don't know how it can be tested because I think it's a case-by-case decision, based on context.

### CODE REVIEW

File: Code Examples/Info Tab Example.nlogo [hunk 1]
```diff
@@ -269,32 +303,52 @@ To include a short piece of code in a sentence, surround it with backticks (`).
...
+```
 
 #### Formatted
 
 You can create a single turtle with the `crt 1` command.
 
 ## Code blocks
 
-It is also possible to have blocks of code. To create a code block, indent every line of the block by 4 spaces.
+It is also possible to have blocks of code. To create a code block, indent every line of the block by 4 spaces. Another way is to sorround it with a three back-ticks before and after the block. (If you don't want your code to be colored as NetLogo code, add `text` after the first three back-ticks.)
```
Comment (nicolaspayette) [inline Code Examples/Info Tab Example.nlogo]: - [ ] Typo: "sorround".
- [ ] "back-tick" should be one word: "backtick"
- [ ] "a three back-ticks": remove the "a"

**Comment by nicolaspayette** [comment]
> I don't know how it can be tested because I think it's a case-by-case decision, based on context.

The goal of a test would be to make sure that:

- You haven't missed any cases.
- Future modelers don't forget to add `text` after the backticks for non-NetLogo code blocks.

How about a test that all four space indented blocks are syntactically valid NetLogo code?

I think we have the means to do that reasonably easily, and it would have value beyond the highlighting question.

File: Code Examples/Info Tab Example.nlogo (hunk 1 above)
Comment (Idloj) [inline Code Examples/Info Tab Example.nlogo]: >  "a three back-ticks": remove the "a"

My intention was "a three backticks line", is it OK?

**Comment by Idloj** [comment]
> How about a test that all four space indented blocks are syntactically valid NetLogo code?

How can that be checked? Trying to parse it is not an option because the code example might be a replacement for some line found in the code, so things like local `let` variables will make the code non-compilable.

**Comment by SethTisue** [comment]
> How can that be checked?

I think the parser has a "forgiving" mode where it just assumes that any identifiers it doesn't recognize are globals?

File: Code Examples/Info Tab Example.nlogo (hunk 1 above)
Comment (nicolaspayette) [inline Code Examples/Info Tab Example.nlogo]: Yes, that works.

**Comment by nicolaspayette** [comment]
> I think the parser has a "forgiving" mode where it just assumes that any identifiers it doesn't recognize are globals?

I think that would be [`checkSyntax`](https://github.com/NetLogo/NetLogo/blob/34d177a051b642aec0a7b16a1a00506552bb0dfe/netlogo-gui/src/main/compiler/Compiler.scala#L79).

I'm not sure if it is actually practical to apply it to code snippets in the info tab, but it would have tremendous value for us if it worked. Do you mind giving it a shot, @Idloj ? Don't waste too much time on it if it seems like you're getting nowhere, but I think it's worth a try.

**Comment by Idloj** [comment]
Done! Sorry for the long delay.

I actually found two real failing cases with the test! So we can say it works quite well (not a 100% of course, but very satisfying IMO).

One of them has a trivial error that I'll fix in a separate commit, but for the other I would like some feedback.
The error is this:
```
HUBNET-SEND expected 3 inputs, a string or list, a string and any input.
hubnet-send "location" (word xcor " " ycor)
```

@nicolaspayette, is there anything else that needs to be done?

**Comment by nicolaspayette** [comment]
I had to make a few adjustments to make the tests pass, but this is now merged into `master`. Thanks @Idloj !

**Comment by Idloj** [comment]
:grinning: Happy to hear that!

**Comment by Idloj** [comment]
Er... Travis fails to compile: https://travis-ci.org/NetLogo/models/builds/162568369#L508

**Comment by nicolaspayette** [comment]
> Travis fails to compile: https://travis-ci.org/NetLogo/models/builds/162568369#L508
or did I miss something?

I fixed those errors in a different branch, which is now merged into `master`, where all tests pass.

**Comment by Idloj** [comment]
ah, ok

**Pull request merged**